src/napari_plugin_manager/base_qt_package_installer.py

```diff
@@ -171,6 +174,79 @@ def _constraints_file(cls) -> str:
         raise NotImplementedError
 
 
+class UvInstallerTool(AbstractInstallerTool):
+    """Uv installer tool for the plugin manager.
+
+    This class is used to install and uninstall packages using uv.
+    """
+
+    @classmethod
+    def executable(cls) -> str:
+        "Path to the executable that will run the task"
+        if sys.platform == 'win32':
+            path = os.path.join(sys.prefix, 'Scripts', 'uv.exe')
+        else:
+            path = os.path.join(sys.prefix, 'bin', 'uv')
+        if os.path.isfile(path):
+            return path
+        return 'uv'
```

Comment:
Since with the latest change the idea is to check for `uv` on the path, this method definition should be updated? Maybe a call to `shutil.which` or something similar needs to be done too and then the current logic should work as a fallback in case that fails since I think it tries to get the `uv` executable guessing the location from the env/Python interpreter root dir napari was launched?